### PR TITLE
Replace waterpark specific files and content

### PR DESCRIPTION
**Pull request by ShreyasPrasad**
Follow-up from #17, closes #20.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (xujustinj) [inline client/package.json:1]: We can probably nuke this file altogether and add it to the `.gitignore`. Also make sure to run `npm i` after editing the packages.

File: server/src/setup/http/express/index.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export { APIRouter, WebServer, WaterparkExpress } from './types'
+export { APIRouter, WebServer, AuthExpress as AuthExpress } from './types'
```
Comment (xujustinj) [inline server/src/setup/http/express/index.ts]:
```suggestion
export { APIRouter, WebServer, AuthExpress } from './types'
```

**Review by xujustinj — COMMENTED** [review]
Compared to #17 we're keeping a lot of stuff around. Are we sure there's nothing further to remove? (e.g. the Waterpark logo SVG if it hasn't been removed already)

**Comment by ShreyasPrasad** [comment]
We'll handle frontend cleanup removal when we get there task wise - otherwise we might be doing extra work reintroducing things we remove imo.

File: client/package.json (hunk 1 above)
Comment (ShreyasPrasad) [reply client/package.json:1]: Why, won't we need it for the auth frontend?

Comment (xujustinj) [reply client/package.json:1]: Oops, mixed it up with `package-lock.json`. It's been a long week

**Pull request merged**